src/component/addnote.js

```diff
@@ -1,88 +1,85 @@
 import "../styles.css";
 import { useNotes } from "../context/noteProvider";
-import { v4 as uuid } from "uuid";
-import {Link} from "react-router-dom";
+// import {Link} from "react-router-dom";
+import dayjs from "dayjs";
+import { BsPin } from "react-icons/bs";
+import { MdPushPin } from "react-icons/md";
+import { Toasters } from "./toasters";
 
 export default function AddNote() {
-  const { state ,dispatch } = useNotes();
+  const { state ,dispatch, addNewNoteHandler } = useNotes();
 
   const labels = ["Home" , "Work" , "Study" , "Gym" , "Health"];
   const priority = ["High" , "Medium" , "Low"]
+  const colors = ["Blue" ,"Green" ,"Yellow", "Purple", "Orange"];
 
   return (
     <div className="AddNote">
-      <div className="add-note-container">
+      <form className="add-note-container" 
+      onSubmit={ (e) => {
+      e.preventDefault()
+      dispatch({ type: "SET_DATE", payload: dayjs().format("YYYY-MM-DD") })
+      addNewNoteHandler(state)
+      dispatch({ type: "CLEAR_NOTE" });
+      }}>
```

Comment:
Suggestion: can extract code from here in separate function and use that function here